src/index.js

```diff
@@ -0,0 +1,158 @@
+import './style.css';
+import Firsticon from './images/refresh-cw.svg';
+import Secondicon from './images/corner-down-left.svg';
+import Thirdicon from './images/more-vertical.svg';
+import Fourthicon from './images/trash-2.svg';
+import {
+  toDoList, refreshDiv, enterDiv, refreshImg, enterImg, listSort, addInput, listWindow, clearAll,
+} from './modules/variables.js';
+
+refreshImg.src = Firsticon;
+enterImg.src = Secondicon;
+refreshDiv.appendChild(refreshImg);
+enterDiv.appendChild(enterImg);
+
+function Options(items) {
+  items.forEach((n) => {
+    n.addEventListener('click', () => {
+      const remover = n.previousElementSibling;
+      remover.classList.toggle('active');
+    });
+  });
+}
+
+class Manager {
+  constructor() {
+    this.toDoList = toDoList;
+  }
+
+  listAddition() {
+    if (!this.toDoList) {
+      this.toDoList = [];
+    }
+    const entry = {
+      description: addInput.value,
+      index: this.toDoList.length,
+      complete: false,
+    };
+    this.toDoList = [...this.toDoList, entry];
+    localStorage.setItem('compiled', JSON.stringify(this.toDoList));
+    this.listRender();
+  }
+
+  listRender() {
+    let lists = '';
+    const sortedList = listSort(this.toDoList);
+    sortedList.forEach((entry) => {
+      let done;
+      let checked;
+      if (entry.complete === true) {
+        done = 'done';
+        checked = 'checked';
+      } else {
+        done = '';
+        checked = '';
+      }
+
+      lists
+      += `<div class="entry">
+        <div>
+        <input type="checkbox" class="boxes" ${checked} name="checkbox" id="${entry.index}">
+        <input type="text" size="125" class="detail ${done}" id="${entry.index}" value="${entry.description}"></input>
+        </div>
+        <img src='${Fourthicon}' class='trash-icon' id='${entry.index}'>
+        <img src='${Thirdicon}' class='dots' id='${entry.index}'>
+      </div>`;
+    });
+    listWindow.innerHTML = lists;
+    const moreIcon = document.querySelectorAll('.dots');
+    Options(moreIcon);
+    const disposeBtn = document.querySelectorAll('.trash-icon');
+    this.listRemoval(disposeBtn);
+    const details = document.querySelectorAll('.detail');
+    this.update(details);
+    const status = document.querySelectorAll('.boxes');
+    this.mark(status);
+  }
+
+  listRemoval(disposeBtn) {
+    disposeBtn.forEach((btn) => {
+      btn.addEventListener('click', () => {
+        const index = btn.getAttribute('id');
+        this.toDoList.splice(index, 1);
+        if (this.toDoList.length > 0) {
+          this.toDoList.forEach((entry, i) => {
+            entry.index = i;
+          });
+        }
+        localStorage.setItem('compiled', JSON.stringify(this.toDoList));
+        this.listRender();
+      });
+    });
+  }
+
+  update(details) {
+    details.forEach((text) => {
+      text.addEventListener('keyup', (e) => {
+        if (e.key !== 'Enter' || !text.value) return;
```

Comment:
I think a read an article earlier that says negative conditions are not good JavaScript best practices

Reply:
Thank you for this reminder Victor. Yes, I've reread some materials and confirmed this. I've modified the functions and removed the negative conditions